src/actions/sources/clearSelectedSource.js

```diff
@@ -0,0 +1,9 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
+
+import type { Action } from "../types";
+
+export const clearSelectedSource: Action = () => ({
+  type: "CLEAR_SELECTED_SOURCE"
+});
```

Comment:
Maybe we can move this to a common file with other functions? so we don't end up with many single files with only one function in them.

Reply:
Great idea, how would you name it?